pkg/apis/azuredisk/v1alpha1/types.go

```diff
@@ -200,3 +221,81 @@ type AzDriverNodeList struct {
 
 	Items []AzDriverNode `json:"items"`
 }
+
+type VolumeCapability struct {
```

Comment:
If this is same struct from the `github.com/container-storage-interface/spec/lib/go/csi` package, we should import it instead, unless there are specific reasons to redefine this struct.

Reply:
Yep, its the same one. I think we are trying to avoid depending on the csi package according to Ed